ci-operator/step-registry/ipi/conf/aws/usage-info/ipi-conf-aws-usage-info-commands.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+#set -o nounset
+#set -o errexit
+#set -o pipefail
+
+set +e
+
+env > "${ARTIFACT_DIR}/prow-ci-env.log"
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+cp ${CLUSTER_PROFILE_DIR}/pull-secret /tmp/pull-secret
+oc registry login --to /tmp/pull-secret
+ocp_version=$(oc adm release info --registry-config /tmp/pull-secret ${RELEASE_IMAGE_LATEST} -ojsonpath='{.metadata.version}' | cut -d. -f 1,2)
+ocp_major_version=$( echo "${ocp_version}" | awk --field-separator=. '{print $1}' )
+ocp_minor_version=$( echo "${ocp_version}" | awk --field-separator=. '{print $2}' )
+rm /tmp/pull-secret
+
+job_type="${JOB_TYPE:-}"
+user=""
+pull_number="null"
+if [[ "${job_type}" == "presubmit" ]]; then
+    user=$(echo "${JOB_SPEC:-}" | jq -r '.refs.pulls[].author')
+    pull_number=${PULL_NUMBER:-unknown}
+elif [[ "${job_type}" == "periodic" ]]; then
+    user=$(echo "${JOB_SPEC:-}" | jq -r '.extra_refs[]' |  jq -r '.repo + "-" + .base_ref')
+else
+    user="null"
```

Comment:
```suggestion
    user=""
```

better an empty string on the pandas side. user will always be a string column.

the installer should not complain about the empty value.

Reply:
done.